source/jormungandr/tests/journey_common_tests.py

```diff
@@ -963,48 +963,6 @@ def test_datetime_error(self):
             assert error["message"] == "Unable to parse datetime, Not naive datetime (tzinfo is already set)"
             assert error["id"] == "unable_to_parse"
 
-    def test_journeys_without_show_codes(self):
```

Comment:
those have been removed because we don't care about show_codes anymore and this was causing problems because in the dataset there and a stop_point AND a stop_area called 'A', and since we do now an autocomplete query we had the stop_area as the from